backend/api.py

```
@@ -52,7 +56,13 @@ def get():
 # TODO: need to commit after executing insert
 @app.route('/insert')
 def insert():
-    validate_user(request.values.get('id_token'))
+    try:
+        validate_user(request.values.get('id_token'))
+    except UserDeniedException as e:
```

This exception will be silenced without any indication that something went wrong.

---

This is repeated below